CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# dbt_shopify v0.12.1
+
+[PR #81](https://github.com/fivetran/dbt_shopify/pull/81) includes the following updates: 
+
+## 🪲 Bug Fixes 🪛
+- Added support for a new `delayed` fulfillment event status from Shopify. This produces a new `count_fulfillment_delayed` field in the `shopify__daily_shop` model.
+
```

Comment:
I'd add your consistency/integrity tests as Under the Hood and that can be the #82 changes.

Reply:
added!